Sources/MapboxCoreNavigation/SDKVersionProvider.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+
+struct VersionProvider {
+    enum MapboxSDK {
+        case mapboxNavigation
```

Comment:
There is a separation between versions of `MapboxNavigation` and `MapboxCoreNavigation` because these versions are used separately in `URLSession.swift` below.

We can consider removing this separation and always assume that these versions are the same.

Reply:
Removed this separation. Now we assume that MapboxCoreNavigation is always the same version as MapboxNavigation